tests/rados/test_stretch_osd_serviceability_scenarios.py

```diff
@@ -0,0 +1,332 @@
+    pool_stat = rados_obj.get_cephdf_stats(pool_name=pool_name)
+    log.debug(pool_stat)
+    post_osd_rm_objs = pool_stat["stats"]["objects"]
+
+    # Objects should be more than the initial no of objects
+    if post_osd_rm_objs <= init_objects:
+        log.error(
+            "Write ops should be possible, number of objects in the pool has not changed"
+        )
+        raise Exception(f"Pool {pool_name} has {pool_stat['stats']['objects']} objs")
+    log.info(
+        f"Successfully wrote {pool_stat['stats']['objects']} on pool {pool_name} in degraded mode\n"
+        f"Proceeding to bring up the nodes and recover the cluster from degraded mode"
+    )
+    log.info("Completed OSD daemon replacement scenarios")
+
+    log.info("Proceeding to do OSD host replacement in Stretch mode")
+    test_host_site = dc_1_name if test_host.hostname in dc_1_hosts else dc_2_name
+    log.info(
+        f"OSD host to be removed : {test_host.hostname} from site {test_host_site}"
+    )
+
+    log.debug(
+        f"Ceph osd tree before host removal : \n\n {rados_obj.run_ceph_command(cmd='ceph osd tree')} \n\n"
+    )
+    try:
+        service_obj.remove_custom_host(host_node_name=test_host.hostname)
+    except Exception as err:
+        log.error(f"Could not remove host : {test_host.hostname}. Error : {err}")
+        raise Exception("Host not removed error")
+    time.sleep(10)
+
+    # Waiting for recovery to post OSD host removal
+    if not rados_obj.check_inactive_pgs_on_pool(pool_name=pool_name):
+        log.error(f"Inactive PGs found on pool : {pool_name}")
+        inactive_pgs += 1
+    method_should_succeed(wait_for_clean_pg_sets, rados_obj, timeout=12000)
+    method_should_succeed(rados_obj.run_pool_sanity_check)
+    if inactive_pgs > 10:
+        log.error("Found inactive PGs on the cluster during OSD removal/Addition")
+        raise Exception("Inactive PGs during stop error")
+
+    # Checking if the expected health warning about the different site weights is displayed post removal of OSD host
+    if not check_stretch_health_warning():
+        log.error("Warnings is not displayed on the cluster post osd host removal")
+        raise Exception("Warning not present on cluster")
+
+    log.debug(
+        f"Ceph osd tree after host removal : \n\n {rados_obj.run_ceph_command(cmd='ceph osd tree')} \n\n"
+    )
+    # Adding a new host to the cluster
+    try:
+        service_obj.add_new_hosts(
+            add_nodes=[test_host.hostname],
+            crush_bucket_name=test_host.hostname,
+            crush_bucket_type=stretch_bucket,
+            crush_bucket_val=test_host_site,
+        )
+    except Exception as err:
+        log.error(
+            f"Could not add host : {test_host.hostname} into the cluster and deploy OSDs. Error : {err}"
+        )
+        raise Exception("Host not added error")
+    time.sleep(60)
+
+    # Waiting for recovery to post OSD host removal
```

Comment:
Minor comment in section - Waiting for recovery to post OSD host addition

Reply:
Updated the comment.